Makefile

```diff
@@ -16,9 +16,9 @@ GIT_SHA=$(shell git rev-parse HEAD)
 GOFLAGS=-ldflags "-X github.com/edgexfoundry/device-modbus-go.Version=$(VERSION)"
 
 build: $(MICROSERVICES)
-	$(GOCGO) build ./...
```

Comment:
why do we need to remove this line?  If we need to, should we also remove from 
https://github.com/edgexfoundry/device-virtual-go/pull/204/files

Reply:
@cloudxxx8,  It causes duplicate build. Yes, I thought I had removed those when I added the `go mod tidy`.

Reply:
it is still there
https://github.com/edgexfoundry/device-virtual-go/blob/702ee4ef1a6f7523fa240909d56a13b89c8d46ac/Makefile#L18